pkgs/applications/video/mpv/wrapper.nix

```
@@ -80,7 +80,7 @@ let
         makeWrapper "${mpv}/Applications/mpv.app/Contents/MacOS/mpv" "$out/Applications/mpv.app/Contents/MacOS/mpv" ${mostMakeWrapperArgs}
       '';
 
-      meta.mainProgram = "mpv";
+      meta = mpv.meta // { mainProgram = "mpv"; };
```

It seems that you can't simply do this because:

```
evaluation aborted with the following error message: 'Failed to evaluate mpv-with-scripts-0.34.0: «broken-outputs»: has invalid meta.outputsToInstall'
```

Probably inheriting the metas from `mpv` is better:
```suggestion
      meta = {
        inherit (mpv.meta) homepage description longDescription maintainers;
        mainProgram = "mpv";
      };
```

I explicitly didn't inherit `license` or `platforms` since it doesn't make too much sense in a wrapper context.